mysql-test/suite/innodb/t/percona_bug_1634932.test

```diff
@@ -0,0 +1,50 @@
+--echo #
```

Comment:
Please use upstream bug number for upstream bugs (ie. bug83648.test). After the Percona Server merge the next step is to send the bugfix to Oracle (get your Percona OCA sorted out meanwhile)